app/controllers/containers/steps_controller.rb

```diff
@@ -49,9 +49,9 @@ def create_container
       else
         @docker_container_wizard_states_environment = @state.environment
         process_error(
-            :error_msg => service.errors,
-            :object => @state.environment,
-            :render => 'environment')
+          :error_msg => service.errors,
+          :object => @state.environment,
+          :render => 'environment')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.